### PR TITLE
New SAT layouts for Chile

### PR DESCRIPTION
**Pull request by smsimon**
Phase I design for Chile SAT layouts using s4aggr+HWP design. Includes: Updated wafer layouts including pin holes, updated clocking for new wafer layouts in focal plane, fix for non-centered focal plane layout, and matched numbers of optically "good" detectors.

Still needs noise information and beams for Chile SAT design.

### CODE REVIEW

File: s4sim/hardware/config.py [hunk 1]
```diff
@@ -1747,14 +1747,14 @@ def sim_nominal():
 
     #from the DSR
```
Comment (keskitalo) [inline s4sim/hardware/config.py:1748]: The old and the new values cannot both be from DSR, can they?

File: s4sim/hardware/sim.py [hunk 2]
```diff
@@ -417,7 +417,7 @@ def sim_telescope_detectors(hw, tele, tubes=None):
 
             # Simulate the offset layout
             offset = qa.mult(
-                qa.rotation(XAXIS, -np.radians(waferspace * platescale / 2)),
+                qa.rotation(XAXIS, -np.radians(waferspace * platescale / np.sqrt(3))),
```
Comment (keskitalo) [inline s4sim/hardware/sim.py:420]: nice catch!

File: s4sim/hardware/config.py [hunk 3]
```diff
@@ -470,59 +470,59 @@ def sim_nominal():
...
+    bnd["pwv_poly"] = 0.930327, 0.068508, 0.001668
     bands["SAT_f095"] = bnd
 
     bnd = OrderedDict()
     bnd["center"] = 155.0
     bnd["low"] = 138.0
     bnd["high"] = 172.1
     bnd["bandpass"] = ""
-    bnd["NET"] = 307.0
+    bnd["NET"] = 365.0
```
Comment (keskitalo) [inline s4sim/hardware/config.py:518]: Big hit :(

**Review by keskitalo — APPROVED** [review]

File: s4sim/hardware/config.py (hunk 1 above)
Comment (smsimon) [reply s4sim/hardware/config.py]: That is a very stale comment--I just updated and deleted it

**Pull request merged**